util/types/time.go

```diff
@@ -973,6 +973,11 @@ func ParseDuration(str string, fsp int) (Duration, error) {
 	if err != nil {
 		return ZeroDuration, errors.Trace(err)
 	}
+	// Invalid TIME values are converted to '00:00:00'
+	// See https://dev.mysql.com/doc/refman/5.7/en/time.html
+	if minute >= 60 || second >= 60 {
+		return ZeroDuration, errors.Trace(ErrInvalidTimeFormat)
```

Comment:
just `return ZeroDuration, ErrInvalidTimeFormat`

Reply:
what about `hour >= 24`?

Reply:
@winkyao 
See https://dev.mysql.com/doc/refman/5.7/en/time.html

MySQL retrieves and displays TIME values in 'HH:MM:SS' format (or 'HHH:MM:SS' format for large hours values). TIME values may range from '-838:59:59' to '838:59:59'. The hours part may be so large because the TIME type can be used not only to represent a time of day (which must be less than 24 hours), but also elapsed time or a time interval between two events (which may be much greater than 24 hours, or even negative).